### PR TITLE
District Mapping - JS changes

### PR DESCRIPTION
**Pull request by gitumarkk**
District/Sector mappings are now stored in the Django backend. This PR covers the connection to the backend to retrieve the hierarchy. That hierarchy should then be converted to array for validation as before. The additional functionality is to watch out for sectors that are not unique and if the user enters one of those, inject an additional question (and logging) into the registration flow.

### CODE REVIEW

File: js_sandbox/lib/go-nike-ghr.js [hunk 1]
```diff
@@ -463,6 +463,15 @@ function GoNikeGHR() {
         return im.config.sectors.indexOf(sector.toLowerCase()) != -1;
     };
 
+    /*************************************************************************************/
```
Comment (imsickofmaps) [inline js_sandbox/lib/go-nike-ghr.js]: Please remove

File: js_sandbox/test/test-go-nike-ghr.js [hunk 2]
```diff
@@ -1250,4 +1298,10 @@ describe("When using the USSD line", function() {
         });
     });
 
+    describe("", function(){
```
Comment (imsickofmaps) [inline js_sandbox/test/test-go-nike-ghr.js]: Remove

**Comment by imsickofmaps** [comment]
@smn ready for review

**Comment by smn** [comment]
Can I get a description on this PR to give me some context of what's being done here?

**Comment by imsickofmaps** [comment]
@smn added. Let me know if not enough detail.

File: js_sandbox/test/fixtures/hierarchy_sectors.json [hunk 3]
```diff
@@ -0,0 +1,5 @@
...
...sizi", "resource_uri": "/api/v1/district/29/"}}, {"id": 402, "name": "Rwimbogo", "resource_uri": "/api/v1/sector/402/", "sector_district": {"id": 29, "name": "Rusizi", "resource_uri": "/api/v1/district/29/"}}, {"id": 403, "name": "Boneza", "resource_uri": "/api/v1/sector/403/", "sector_district": {"id": 30, "name": "Rutsiro", "resource_uri": "/api/v1/district/30/"}}, {"id": 404, "name": "Gihango", "resource_uri": "/api/v1/sector/404/", "sector_district": {"id": 30, "name": "Rutsiro", "resource_uri": "/api/v1/district/30/"}}, {"id": 405, "name": "Kigeyo", "resource_uri": "/api/v1/sector/405/", "sector_district": {"id": 30, "name": "Rutsiro", "resource_uri": "/api/v1/district/30/"}}, {"id": 406, "name": "Kivumu", "resource_uri": "/api/v1/sector/406/", "sector_district": {"id": 30, "name": "Rutsiro", "resource_uri": "/api/v1/district/30/"}}, {"id": 407, "name": "Manihira", "resource_uri": "/api/v1/sector/407/", "sector_district": {"id": 30, "name": "Rutsiro", "resource_uri": "/api/v1/district/30/"}}, {"id": 408, "name": "Mukura", "resource_uri": "/api/v1/sector/408/", "sector_district": {"id": 30, "name": "Rutsiro", "resource_uri": "/api/v1/district/30/"}}, {"id": 409, "name": "Murund
```
Comment (smn) [inline js_sandbox/test/fixtures/hierarchy_sectors.json:4]: fixture looks odd, are you sure this is correct? Also the trailing comma in the JSON would make it fail in Node.

**Comment by smn** [comment]
:+1:

File: js_sandbox/test/fixtures/hierarchy_sectors.json (hunk 3 above)
Comment (gitumarkk) [inline js_sandbox/test/fixtures/hierarchy_sectors.json:4]: @smn the tests were passing so I assumed its correct.

**Pull request merged**